recipes/libtins/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conans import tools, CMake, ConanFile
+import os
+
+
+class LibTinsConan(ConanFile):
+    name = "libtins"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mfontanini/libtins"
+    description = "High-level, multiplatform C++ network packet sniffing and crafting library."
+    license = "BSD-2-Clause"
+    topics = ("pcap", "packets", "network", "packet-analyser", "packet-parsing", "libpcap", "sniffing")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_ack_tracker": [True, False],
+        "with_wpa2": [True, False],
+        "with_dot11": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_ack_tracker": True,
+        "with_wpa2": True,
+        "with_dot11": True,
+    }
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("libpcap/1.10.0")
+        if self.options.with_ack_tracker:
+            self.requires("boost/1.75.0")
+        if self.options.with_wpa2:
+            self.requires("openssl/1.1.1i")
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+        tools.replace_in_file(
+                os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                "OPENSSL",
+                "OpenSSL"
+        )
+        tools.replace_in_file(
+                os.path.join(self._source_subfolder, "src", "CMakeLists.txt"),
+                "OPENSSL",
+                "OpenSSL"
+        )
```

Comment:
Not like this.
Replace sparingly (`OPENSSL_FOUND`, `OPENSSL_LIBRARIES` and `OPENSSL_INCLUDE_DIR`) and not in `source()` but in `build()`

Reply:
Patching might be a better approach (it might be not).

Also, please create a PR in the upstream project to make this change redundant in the future.

Reply:
No, there is nothing to submit to upstream, it's an issue in cmake_find_package generator.

Reply:
Oh, that's unfortunate, because I think there are many cases like this in the CCI.

Reply:
https://github.com/conan-io/conan-center-index/pull/4597 should fix this issue